libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -199,7 +209,6 @@ export class BotFrameworkAdapter
     // These keys are public to permit access to the keys from the adapter when it's a being
     // from library that does not have access to static properties off of BotFrameworkAdapter.
     // E.g. botbuilder-dialogs
-    public readonly ConnectorClientKey: symbol = Symbol('ConnectorClient');
```

Comment:
Pushed up to the `BotAdapter`, again not sure that's right but it's useful in the Cloud Adapter implementation.